galley/pkg/config/analysis/analyzers/util/config.go

```diff
@@ -39,3 +39,7 @@ func MeshConfig(ctx analysis.Context) *v1alpha1.MeshConfig {
 func IsSystemNamespace(ns resource.Namespace) bool {
 	return ns == "kube-system" || ns == "kube-public"
 }
+
+func IsIstioSystemNamespace(ns resource.Namespace) bool {
+	return ns == "istio-system"
+}
```

Comment:
Note that the "istio system" namespace may be something else. I believe you can even end up with configurations where Istio doesn't have a specialized namespace at all. A more accurate way to think about it is that we want to skip the Istio control plane. 

Instead of hardcoding `istio-system` here and checkin the namespace, we should look at the resource label for `istio=` as a marker for control plane resources. (See [this](https://github.com/istio/istio/pull/19274#discussion_r351559524) thread that discusses the reasoning for doing so). You should be able to refactor out the [existing code](https://github.com/istio/istio/blob/master/galley/pkg/config/analysis/analyzers/auth/mtls.go#L115-L120) for that into a shared util function.